src/components/Views/CameraMainView/CameraMainView.js

```diff
@@ -80,6 +82,27 @@ class CameraMainView extends React.Component<
     loading: false
   };
 
+  componentDidMount() {
```

Comment:
probably want to do
`async componentDidMount() { await PermissionsAndroid.request(...) }`